plugins/woocommerce/client/legacy/css/twenty-twenty-one.scss

```
@@ -3050,6 +3050,20 @@ a.reset_variations {
 	}
 }
 
+@media only screen and (max-width: 768px) {
+	.woocommerce-message,
+	.woocommerce-error li,
+	.woocommerce-info {
+		padding: 1rem 1.5rem;
+
+		.button {
```

I think that if we increase the specificity of this css rule, we can get rid of the `!important` statement on line 3062, for example, replacing: `.button` with `a.button { }`
```suggestion
		a.button {
```

---

Ah, nice catch. This does make the selector more specific and negates the need for `!important`